llama_hub/file/pandas_excel/base.py

```diff
@@ -36,11 +36,12 @@ def __init__(
         super().__init__(*args, **kwargs)
         self._pandas_config = pandas_config
         self._concat_rows = concat_rows
-        self._row_joiner = row_joiner
+        self._row_joiner = row_joiner if row_joiner else "\n"
 
     def load_data(
         self,
         file: Path,
+        pandas_config: dict = {},
```

Comment:
it's not safe to initialize to {} in fn signature, conventionally what we do is do pandas_config: Optional[dict] = None, and then in the function def do something like pandas_config = pandas_config or {}